apps/slack/mac.py

```diff
@@ -9,6 +9,10 @@
 
 @ctx.action_class("user")
 class UserActions:
+    def slap():
```

Comment:
What is the conceptual action that user.slap() is actually encapsulating here? It ought to be named after that instead of something ambiguous like "slap". Also it needs an accurate docstring.

Reply:
Yeah, good call. And nice catch on the docstring.